guides/common/assembly_managing-red-hat-subscriptions.adoc

```diff
@@ -10,6 +10,8 @@ include::modules/proc_removing-red-hat-subscriptions-and-subscription-manifests.
 
 include::modules/proc_updating-and-refreshing-red-hat-subscription-manifests.adoc[leveloffset=+1]
 
+include::modules/proc_configuring-subscriptions-service.adoc[leveloffset=+1]
```

Comment:
I'm not entirely sure where this information should go in the context of this chapter. I'm not saying this placement is wrong, though, it just feels off. I would see it either as the first procedure or the last module in the chapter. Still, not sure how it fits.

Reply:
I'll move it to be the first procedure.